streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -365,7 +364,7 @@ public GroupAssignment assign(final Cluster metadata, final GroupSubscription gr
             // the maximum of the depending sub-topologies source topics' number of partitions
             final Map<Integer, TopicsInfo> topicGroups = taskManager.builder().topicGroups();
 
-            final Map<TopicPartition, PartitionInfo> allRepartitionTopicPartitions = prepareRepartitionTopics(topicGroups, metadata);
+            final Map<TopicPartition, PartitionInfo> allRepartitionTopicPartitions = prepareRepartitionTopics(metadata);
```

Comment:
nit: line 365 above can be moved down now since it is only needed before line 379.